logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -338,8 +350,73 @@ actual class CallManagerImpl(
         return AvsCallBackError.NONE.value
     }
 
+    /**
+     * onCallingReady
+     * Will start the handlers for: ParticipantsChanged, NetworkQuality, ClientsRequest and ActiveSpeaker
+     */
+    private fun onCallingReady() {
+        val participantChangedHandler = ParticipantChangedHandlerImpl(
+            participantMapper = callMapper.participantMapper,
+            onParticipantsChanged = { conversationId, participants, clients ->
+                onParticipantsChanged(
+                    conversationId = conversationId,
+                    participants = participants
+                )
+
+                scope.launch {
+                    withCalling {
+                        wcall_request_video_streams(
+                            inst = deferredHandle.await(),
+                            convId = conversationId,
+                            mode = DEFAULT_REQUEST_VIDEO_STREAMS_MODE,
+                            json = clients.toJsonString()
+                        )
+                    }
+                }
```

Comment:
Is it needed for the scope of group calls ?

Reply:
Yes, as new participants join the call we will be requesting the video stream from them, as I understood it to be end-to-end also based on the video stream for each participant and self

Reply:
I think it's required later when we implement video calls..No problem to keep it it should be done anyway ✅